torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4484,6 +4500,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
                SkipInfo('TestCommon', 'test_out', dtypes=[torch.float32]),
            )),
+    OpInfo('logcumsumexp',
+           dtypes=floating_types_and(),
+           dtypesIfCUDA=floating_types_and(torch.half),
+           skips=(
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               SkipInfo('TestCommon', 'test_out', dtypes=(torch.float32,), device_type='cuda'),
```

Comment:
Does this skip really only need to be applied on CUDA?

Reply:
Per offline discussion: yes!

The implementations for cum* ops diverge on CPU and CUDA, and the check is not performed in the CUDA version of the op only.